pyvista/utilities/helpers.py

```diff
@@ -1198,6 +1198,7 @@ def try_callback(func, *args):
         formatted_exception = 'Encountered issue in callback (most recent call last):\n' + ''.join(
             traceback.format_list(stack) + traceback.format_exception_only(etype, exc)
         ).rstrip('\n')
+        1 / 0
```

Comment:
huh?

Reply:
Debug statement, not quite ready for merge.